internal/runner/session.go

```diff
@@ -309,6 +314,11 @@ func (es *owlEnvStorer) addEnvs(envs []string) error {
 	return nil
 }
 
+func (es *owlEnvStorer) getEnv(name string) string {
+	// TODO(adamb): implement this
```

Comment:
@sourishkrout I will need some help here. Is the owl store designed in a way that it's possible to get a single env value by name?

Reply:
Not yet but I can add it later. I'd suggest to use a snapshot for now and leave a todo for me. Something roughly like below should work:

```go
	snapshot, err := es.owlStore.Snapshot()
	for _, item := range snapshot {
		if item.Var.Name == "PATH" {
			...
			break
		}
	}
```